exercises/matrix/canonical-data.json

```diff
@@ -0,0 +1,52 @@
+{
+  "exercise": "matrix",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "one number matrix",
+      "property": "rows",
+      "input": {
+        "string": "1",
+        "index": 0
+      },
+      "expected": [1]
+    },
+    {
+      "description": "can extract row",
+      "property": "rows",
+      "input": {
+        "string": "1 2\n3 4",
+        "index": 1
+      },
+      "expected": [3,4]
+    },
+    {
+      "description": "numbers of different length",
+      "property": "rows",
+      "input": {
+        "string": "1 2\n10 20",
+        "index": 1
+      },
+      "expected": [10,20]
+    },
+    {
+      "description": "can extract column",
+      "property": "columns",
```

Comment:
Seeing how only a single index is being used, shouldn't this be named `column` (singular)?